workspaces/cli-server/src/server.ts

```diff
@@ -98,6 +98,10 @@ class CliServer {
 
     const anonIdPromise = getOrCreateAnonId();
 
+    app.get('/api/health', async (req, res) => {
+      res.json({ status: 'ok' });
```

Comment:
may as well stick the version in here now. i'm curious how often we run into client version being out of sync with daemon version

Reply:
I'll add this in and get sentry to report on errors